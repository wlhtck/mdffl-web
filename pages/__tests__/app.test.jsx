// import React from 'react';
// import { shallow } from 'enzyme';
// import App from '../app';
//
// describe('pages/app', () => {
//   it('renders an app page', () => {
//     expect(shallow(<App router={{ query: { data: { test: 'data' } } }} />)).toMatchSnapshot();
//   });
// });
