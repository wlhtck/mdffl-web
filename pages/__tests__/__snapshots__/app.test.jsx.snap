// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/app renders an app page 1`] = `
<Component
  router={
    Object {
      "query": Object {
        "data": Object {
          "test": "data",
        },
      },
    }
  }
/>
`;
